doc/source/api.rst

```diff
@@ -1567,6 +1576,22 @@ Conversion
    DatetimeIndex.to_pydatetime
    DatetimeIndex.to_series
 
+Period
+------
+
```

Comment:
also these need to go in a new section: Scalars I think (alternatively they could go with there associated Indexes), but I think a Scalars section is fine.